hi-ml-azure/README.md

```diff
@@ -0,0 +1,146 @@
+# Microsoft Health Intelligence Machine Learning Toolbox
```

Comment:
Should we add "Azure" here? Also on line 9 "ML components will be added in the next few weeks" can be deleted

Reply:
Line 20 also refers to this as the hi-ml package:  are we going to refer to this separately as this hi-ml-azure package?